app/models/certification.rb

```diff
@@ -17,21 +17,16 @@ class Certification < ApplicationRecord
   belongs_to :claim, class_name: 'Claim::BaseClaim'
   belongs_to :certification_type
 
+  validates :certification_type_id, presence: true, if: :claim_agfs?
   validates_with CertificationValidator
-  validate :at_least_one_boolean_selected
-
-  acts_as_gov_uk_date :certification_date, error_clash_behaviour: :override_with_gov_uk_date_field_error
 
   def perform_validation?
     true
   end
 
   private
 
-  def at_least_one_boolean_selected
-    return unless claim.is_a?(Claim::AdvocateClaim)
-
-    return if certification_type.present?
-    errors[:base] << 'You must select one option on this form'
+  def claim_agfs?
+    claim.is_a?(Claim::AdvocateClaim)
```

Comment:
Not from this PR, but want to check this business logic as this implies only agfs "final" claims require certification type, which may be true but if so then the check could be `agfs? && final?` or similar i believe.

Reply:
and method should be called `claim_agfs_final?` or similar